crates/viewer/re_view_spatial/tests/visible_time_range.rs

```diff
@@ -344,28 +329,42 @@ fn setup_blueprint(
         let view_id = view_blueprint.id;
         blueprint.add_views(std::iter::once(view_blueprint), None, None);
 
+        // Set the bounds such that the points are fully visible, that way we get more pixels contributing to the output.
+        let property_path = re_viewport_blueprint::entity_path_for_view_property(
+            view_id,
+            ctx.store_context.blueprint.tree(),
+            re_types::blueprint::archetypes::VisualBounds2D::name(),
+        );
+        ctx.save_blueprint_archetype(
+            &property_path,
+            &re_types::blueprint::archetypes::VisualBounds2D::new(re_types::datatypes::Range2D {
+                x_range: [0.0, 100.0].into(),
+                y_range: [0.0, 100.0].into(),
+            }),
+        );
+
         if let Some(time_range) = time_range {
-            let visible_time_range_list = vec![re_types::blueprint::components::VisibleTimeRange(
-                time_range,
-            )];
+            let visible_time_range_list =
+                re_types::blueprint::archetypes::VisibleTimeRanges::new([time_range]);
             let property_path = re_viewport_blueprint::entity_path_for_view_property(
                 view_id,
                 ctx.store_context.blueprint.tree(),
                 re_types::blueprint::archetypes::VisibleTimeRanges::name(),
             );
 
-            ctx.save_blueprint_component(&property_path, &visible_time_range_list);
+            ctx.save_blueprint_archetype(&property_path, &visible_time_range_list);
         }
 
         if let Some(green_time_range) = green_time_range {
-            let visible_time_range_list = vec![re_types::blueprint::components::VisibleTimeRange(
-                green_time_range,
-            )];
-            let property_path = view_id
-                .as_entity_path()
-                .join(&EntityPath::from("ViewContents/recursive_overrides/green"));
-
-            ctx.save_blueprint_component(&property_path, &visible_time_range_list);
+            let visible_time_range_list =
+                re_types::blueprint::archetypes::VisibleTimeRanges::new([green_time_range]);
+            ctx.save_blueprint_archetype(
+                &re_viewport_blueprint::ViewContents::override_path_for_entity(
+                    view_id,
+                    &"green".into(),
+                ),
```

Comment:
So, this tests the individual path, right? Do we have any test for the recursive path?

Reply:
as per comment below. No, because we don't expose it either. It's a remnant of a discarded design and only used for `visible` on some ui interactions